src/content/en/updates/posts/2015/11/app-shell.markdown

```diff
@@ -127,77 +136,79 @@ During the first load experience, your goal is to get meaningful content to the
 
 ## Tooling
 
-We maintain a number of different [service worker helper libraries](https://developers.google.com/web/tools/service-worker-libraries/?hl=en) that make the process of precaching your application’s shell or handling common caching patterns more straight-forward to setup.
+We maintain a number of different [service worker helper libraries](https://developers.google.com/web/tools/service-worker-libraries/?hl=en) that make the process of precaching your application’s shell or handling common caching patterns easier to setup.
 
 ![Screenshot of the Service Worker Library Site on Web Fundamentals]({{site.WFBaseUrl}}/updates/images/2015/11/appshell/image_10.png)
 
-## Use sw-precache for Your Application Shell
+### Use sw-precache for Your Application Shell
 
-Using [sw-precache](https://github.com/GoogleChrome/sw-precache) to cache the application shell should handle the concerns around file revisions, the install/activate questions, and the fetch scenario for the app shell.  You can drop sw-precache into your application’s build process and can use configurable wildcards to pick up your static resources. Rather than manually hand-crafting your service worker script, it generates one for you to take care of cache management in a way that is safe and has an efficient, cache-first fetch handler.
+Using [sw-precache](https://github.com/GoogleChrome/sw-precache) to cache the application shell should handle the concerns around file revisions, the install/activate questions, and the fetch scenario for the app shell.  Drop sw-precache into your application’s build process and use configurable wildcards to pick up your static resources. Rather than manually hand-crafting your service worker script, let sw-precache generate one that manages your cache in a safe and efficient, using a cache-first fetch handler.
 
-Initial visits to your app trigger precaching of the complete set of needed resources. This is very similar to the experience of installing a native app from an app store. When users return to your app, only updated resources are downloaded. In our demo app, we display an "App updated. Refresh for the new version" message to inform users that a new version of the application shell is available. This pattern is a low-friction way of letting users know they can refresh for the latest version.
+Initial visits to your app trigger precaching of the complete set of needed resources. This is similar to the experience of installing a native app from an app store. When users return to your app, only updated resources are downloaded. In our demo, we inform users when a new shell is available with  the message, "App updates. Refresh for the new version." This pattern is a low-friction way of letting users know they can refresh for the latest version.
 
-## Use sw-toolbox for Runtime Caching
+### Use sw-toolbox for Runtime Caching
```

Comment:
Have the docs for how to use sw-toolbox and sw-precache been updated to make this easier?